// Copyright 2013 Google Inc.
//
// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at
//
//      http://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS,
// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
// See the License for the specific language governing permissions and
// limitations under the License.

// Author: huibao@google.com (Huibao Lin)
// Protobufs for storing enum related to logging.

syntax = "proto2";

/*
*/

option optimize_for = LITE_RUNTIME;
/*
*/

package net_instaweb;

enum ImageType {
  IMAGE_UNKNOWN = 0;
  IMAGE_JPEG = 1;
  IMAGE_PNG = 2;
  IMAGE_GIF = 3;
  IMAGE_WEBP = 4;
  IMAGE_WEBP_LOSSLESS_OR_ALPHA = 5; // webp that is lossless or transparent.
  IMAGE_WEBP_ANIMATED = 6;
}
